config/smtp.yml.template

```diff
@@ -0,0 +1,20 @@
+default: &default
+  default_url_options:
+    host: dradis-pro.dev
+  smtp_settings:
+    address:               smtp.gmail.com
```

Comment:
I don't think the default should actually be setup with an address.
If someone runs bin/setup this setup will be used. If they create issues gmail will start receiving un-authenticated mail attempts. Which would leak comments/issue information into their logs even if the mail is rejected.